logs_http.module

```diff
@@ -93,146 +68,27 @@ function _logs_http_decode_exception($exception) {
  * devel_boot() registers this function as a shutdown function.
  */
 function logs_http_shutdown() {
-  if (!$events = logs_http_get_registered_events()) {
+  if (!LogsHttpLogger::isEnabled()) {
```

Comment:
ok , but you are using it as a static class. You can get the injected class by something like:

````php
/** @var LogsHttpLoggerInterface $logs_http_logger */
$logs_http_logger = \Drupal::service('logs_http.logs_http_logger');
```